src/main/java/de/mediathekview/mserver/crawler/rbb/tasks/RbbDayTask.java

```diff
@@ -0,0 +1,55 @@
+package de.mediathekview.mserver.crawler.rbb.tasks;
+
+import static de.mediathekview.mserver.base.Consts.ATTRIBUTE_HREF;
+
+import de.mediathekview.mserver.base.utils.UrlUtils;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractRecrusivConverterTask;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.rbb.RbbConstants;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class RbbDayTask extends AbstractDocumentTask<CrawlerUrlDTO, CrawlerUrlDTO> {
+
+  private static final String SELECTOR_ENTRY = "div.box > div.entries > div.entry";
+  private static final String SELECTOR_URL = "a.mediaLink";
+
+  public RbbDayTask(AbstractCrawler aCrawler,
+      ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)